examples/terraform-integrations/terraform-workflows-templates.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
+  annotations:
+    argocd.argoproj.io/sync-wave: "3"
```

Comment:
Why the sync wave?

Reply:
Thats because we have dependencies with Flux, Tofu controllers.

Reply:
Sync waves won't take effect here since this isn't app of apps.